src/digger/digger/digger_node.py

```diff
@@ -101,15 +90,13 @@ def toggle(self, digger_belt_power: float) -> None:
         else:
             self.set_power(digger_belt_power)
 
-    # TODO: This method can probably be deleted during the implementation of ticket #257
     def set_position(self, position: int) -> None:
         """This method sets the position (in degrees) of the digger."""
-        self.current_goal_position = position  # goal position should be in degrees
         self.cli_motor_set.call_async(
```

Comment:
This will not wait unless you await it

Reply:
Fixed with .result() from the python future library